api/Document.json

```diff
@@ -11217,54 +11217,6 @@
           }
         }
       },
-      "undoManager": {
```

Comment:
This removal is correct, it's from https://github.com/WebKit/WebKit/blob/92ce6b62a589a0a70da09a6adfcfa03a07624467/Source/WebCore/dom/Document%2BUndoMananger.idl#L27 but hasn't been shipped.